packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -236,7 +235,7 @@ List<String> _xcodeBuildSettingsLines({
   }
 
   if (buildInfo.dartDefines?.isNotEmpty ?? false) {
-    xcodeBuildSettings.add('DART_DEFINES=${jsonEncode(buildInfo.dartDefines)}');
+    xcodeBuildSettings.add('DART_DEFINES=${buildInfo.dartDefines.join(',')}');
```

Comment:
Is this line the difference between this PR and https://github.com/flutter/flutter/pull/54909?  I played where's Waldo.

Reply:
Yes! sorry :(